typescript/sdk/src/metadata/agentConfig.ts

```diff
@@ -381,6 +381,27 @@ export const ValidatorAgentConfigSchema = AgentConfigSchema.extend({
           ),
       })
       .describe('A checkpoint syncer that uses S3'),
+    z
+      .object({
+        type: z.literal('gcs'),
```

Comment:
Can this utilize the value via the `CheckpointSyncerType` enum instead of duplicating the const here?

Reply:
Resolving this to unblock the merging, if anyone still considers this substantial, let's create another issue about it like I suggested above.